### PR TITLE
AGPLv3+ license

### PR DESCRIPTION
**Pull request by comradekingu**
Needs consent from all contributors to date.

### CODE REVIEW

**Comment by aschrijver** [comment]
Here I will stick to the ways the top-level awesome is set up, and its [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md) strongly recommends using CC0 or another CC license. I will check if there were changes made to how they mention the license, and adjust accordingly.

**Comment by comradekingu** [comment]
Ah, I noticed now that it said CC0 at the bottom.
The good thing is no consent is needed, and the bad thing is that CC0 is functionally void for collaborative projects.
So it isn't public domain, but it does create problems because it isn't. This is because what public domain is, and whether something can be committed to it differs between jurisdictions. They are also incompatible concerns, but then again CC0 is avoided by anything people actually rely on as far as I can tell.
Also CC0 is another reason why "CC" is a meaningless term overall.

And as predicted…
>We strongly recommend the CC0 license, but any Creative Commons license will work.

So NC is OK? If CC0 is good, is a license specifically incompatible with it also good?

>A code license like MIT, BSD, Apache, GPL, is not acceptable.

Neither "BSD", "Apache", or "GPL" is a license. Arguably neither is "MIT", for other unfortunate reasons.

I also don't think the filename should be "license", but alas.

TL;DR The actual license (CC0 here) should be in the appropriately named file, in the root of the repo (otherwise the automated GitHub listing doesn't work).

**Comment by aschrijver** [comment]
Thanks once more. I will stick with CC0, but explicitly adding the license to the repo is a good idea. See #86

**Comment by comradekingu** [comment]
@aschrijver
Btw, the the manifesto says
https://github.com/sindresorhus/awesome/blob/main/awesome.md
>We would recommend CC0. Code licenses like MIT, BSD, GPL, and so forth are not recommended.
